### DISCUSSION TITLE
15GB docker image fails to deploy

### DISCUSSION THREAD

**Discussion by rydes01** [aws/copilot-cli#5799] [Q&A] [answered]
Our docker image has increased in size from under 10GB to 15GB and we cannot deploy using Copilot anymore. When we deploy we get the following error:

```
$ copilot svc deploy
✘ Latest 1 task stopped reason                                                                                  
 - [<hash>]: CannotPullContainerError: ref pull has been retried 1 time(s):
 failed to extract layer <hash>:
 write <path>: no space left on device: unknown       
```

Is it possible to increase the space available? Is there some documented maximum image size?

**Comment by dannyrandall** [comment, marked as answer]
You should be able to use [`storage.epheremal`](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#ephemeral) to increase the amount of storage given to a task.
